modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -216,6 +216,6 @@ This section describes security considerations and known issues.
 
 === Authorization not triggered for empty match
 
-If a query yields no results, the xref::authentication-and-authorization/authorization.adoc[Authorization] process will not be triggered.
+If a query yields no results, the xref::/type-definitions/directives/security/authorization.adoc[Authorization] process will not be triggered.
```

Comment:
Just to double check: you decided to keep the directives folder inside the type definitions one, even though they are separate topics in the content-nav?

Reply:
good catch. i was thinking about removing the type-definitions step in the scope of this PR. i went without it to make it less confusing, but would probably have done it as a follow-up. i might as well do it now though, since we're already discussing it :thinking:

Reply:
ok, this added quite a few files touched to the PR, but it's done